### ISSUE TITLE
libmirage-apple.xml is too broad

### ISSUE THREAD

**Issue by heftig** [cdemu/cdemu#20]
Installing libmirage breaks fwupd. The latter has a quirks file with this content:

`/usr/share/fwupd/quirks.d/redfish.quirk`
```dosini
# Lenovo ThinkSystem
[42f00735-c9ab-5374-bd63-a5deee5881e0]
Flags = wildcard-targets,reset-required

[REDFISH\VENDOR_Lenovo&ID_BMC-Backup]
ParentGuid = REDFISH\VENDOR_Lenovo&ID_BMC-Primary
Flags = dual-image,is-backup

[REDFISH\VENDOR_Lenovo&ID_BMC-Primary]
ParentGuid = REDFISH\VENDOR_Lenovo&ID_BMC-Primary
Flags = dual-image

[REDFISH\VENDOR_Lenovo&ID_LXPM]
Flags = ~updatable
[REDFISH\VENDOR_Lenovo&ID_LXPMLinuxDriver]
Flags = ~updatable
ParentGuid = REDFISH\VENDOR_Lenovo&ID_LXPM
[REDFISH\VENDOR_Lenovo&ID_LXPMWindowsDriver]
Flags = ~updatable
ParentGuid = REDFISH\VENDOR_Lenovo&ID_LXPM
```

Unfortunately, this file happens to be covered by the `<match value="PM" type="string" offset="512"/>`, turning it into `application/x-apple-diskimage` instead of the expected `text/plain`.

**Comment by rokm** [comment]
Thanks for the report! In the retrospect, the two-character patterns used in `libmirage-apple.xml` indeed seem to lack sufficient descriptive power and are likely to generate false positives, so I think I'll remove them.

**Comment by rokm** [comment]
Fixed in 3.2.6.
